plugins/inputs/openweathermap/README.md

```diff
@@ -0,0 +1,145 @@
+# Telegraf Plugin: openweathermap
+
+OpenWeatherMap provides the current weather and forecasts for more than 200,000 cities. To use this plugin you will need a token. For more information [click here](https://openweathermap.org/appid).
+
+Find city identifiers in this [list](http://bulk.openweathermap.org/sample/city.list.json.gz)
+
+### Configuration:
+
+```
+[[inputs.openweathermap]]
+  ## Root url of API to pull stats
+  base_url = "http://api.openweathermap.org/data/2.5/"
+  # Your personal user token from openweathermap.org
+  app_id = "xxxxxxxxxxxxxxxxxxxxxxx"
+  # List of city identifiers
+  cities = ["2988507", "519188"]
+  # HTTP response timeout (default: 5s)
+  response_timeout = "5s"
+```
+
+### Measurements & Fields:
+
+- weather
+  - humidity
+  - temperature
+  - pressure
+  - rain
+  - wind.speed
+  - wind.deg
+
+### Tags:
+
+- weather
+  - server
+  - port
+  - base_url
+  - city_id
```

Comment:
Maybe add `city_name` as a tag

Reply:
I was concerned about homonyms, that's why I did not use city_name as a tag.